### ISSUE TITLE
Question: Does the learning rate need to be adjusted with gpus or batch size?

### ISSUE THREAD

**Issue by damonmaria** [open-mmlab/mmsegmentation#380]
The [docs here](https://mmsegmentation.readthedocs.io/en/latest/train.html#train-a-model) (the "*important*" note) imply to me that the learning rate is affected by the number of gpus and batch size. 

Should I adjust the learning rate if I change the gpus or batch size from the default in a configuration? And if so, how to calculate the adjustment?

**Comment by xvjiarui** [comment]
Hi @damonmaria 
You may use linear scale up rule [here](https://arxiv.org/abs/1706.02677).

**Comment by damonmaria** [comment]
So I gather if we doubled the GPUs, or doubled `samples_per_gpu` then the learning rate should be doubled as well. 

Thanks @xvjiarui.

**Comment by MengHao666** [comment]
> So I gather if we doubled the GPUs, or doubled `samples_per_gpu` then the learning rate should be doubled as well.
> 
> Thanks @xvjiarui.

Hi , are u sure that the scale  should be linear with the GPUs*samples_per_gpu?

**Comment by damonmaria** [comment]
@MengHao666 After some trial and error and further investigation, the batch size has other implications for training and is often dependent on your own dataset. I'd recommend trying different batch sizes and learning rates to see what works best for you. But the above 'rule of thumb' provides a good starting point.
